package.json

```diff
@@ -74,7 +74,7 @@
     "jest-environment-jsdom": "^28.1.0",
     "ts-loader": "^8.4.0",
     "typescript": "^4.0.3",
-    "webpack": "^5.73.0",
+    "webpack": "^5.76.0",
```

Comment:
since based on the package-lock, it looks like we've upgraded to v5.90.0, can we update this value to match that? it makes it easier to keep track